os/include/tinyara/lcd/st7785.h

```diff
@@ -69,22 +71,22 @@
 static const lcm_setting_table_t lcd_init_cmd_g[] = { 
         {0x11, 0, {0x00}},
         {REGFLAG_DELAY, 120, {}},       /* Delayms (120) */
-        {0x36, 2, {0x00, LCD_ORIENTATION}},
```

Comment:
Could you please revert st7785.h changes?

Reply:
done~